playbooks/byo/openshift-cluster/upgrades/v3_1_to_v3_2/upgrade.yml

```diff
@@ -4,10 +4,6 @@
   become: no
   gather_facts: no
   tasks:
-  - name: Verify Ansible version is greater than or equal to 1.9.4 and less than 2.0
```

Comment:
I think I removed this as something else was replicating this check, IIRC openshift_facts but I can check tomorrow.